pkg/olm/olm.go

```diff
@@ -217,8 +217,78 @@ func GenerateCSV(opConf *operator.Conf) *operv1.ClusterServiceVersion {
 			operv1.SpecDescriptor{Path: "tolerations", XDescriptors: []string{"urn:alm:descriptor:io.kubernetes:Tolerations"}},
 			operv1.SpecDescriptor{Path: "imagePullSecret", XDescriptors: []string{"urn:alm:descriptor:io.kubernetes:Secret"}},
 		},
-		"BackingStore":      []operv1.SpecDescriptor{},
-		"BucketClass":       []operv1.SpecDescriptor{},
+		"BackingStore": []operv1.SpecDescriptor{
+			operv1.SpecDescriptor{Description: "Region is the AWS region.",
+				Path:         "awsS3.region",
+				XDescriptors: []string{"urn:alm:descriptor:com.tectonic.ui:awsS3", "urn:alm:descriptor:com.tectonic.ui:text"},
```

Comment:
These x-descriptors are textual constants that repeat multiple times.
What do you think about defining them as const strings and use them to avoid typos and mistakes?

Reply:
@guymguym I have resolved this in https://github.com/noobaa/noobaa-operator/pull/162.

Reply:
I accidentally deleted my branch and so affected this PR(closed) and cannot re-open it